src/main/java/com/example/name/NameResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.example.name;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class NameResponse {
+    private final Map<String, String> response;
```

Comment:
このようにjsonのキーをフィールド名として定義すると良いですよ。
```suggestion
    private final String message;
```

Reply:
message フィールドに変更し、単一のメッセージを表すように修正致しました。
コミットハッシュ値
871b63dfdebd51a333551b7740088cda99944af0
また、上記に伴いcontrollerクラスのDeleteメソッドについても修正致しました。
e37485c08c01a64c9ce141f6c4ddc1dd8d7bba16

実行結果
![image](https://github.com/yuuksibunta/Final-Assignment/assets/148073110/cd9bbc75-fe5a-467d-a8d7-540ad5dfe474)

Reply:
okです